atest/testdata/standard_libraries/string/get_matching_lines.robot

```diff
@@ -64,6 +67,27 @@ Get Lines Matching Regexp With Case-Insensitive
     Test Get Lines Matching Regexp    ${INPUT}    (?i).*LINE    Third line
     Test Get Lines Matching Regexp    ${INPUT}    .*LINE.*    ${EMPTY}
 
+Get Lines Containing Regexp Matching Two Words
+    Test Get Lines Containing Regexp    ${INPUT}    more|row    One more row here
+
```

Comment:
Please note the ${EMPTY} result when exact_match=true. It is one of the problematic examples. I added more examples to documentation to illustrate this.